packages/blocks/src/api/parser.js

```diff
@@ -454,6 +454,11 @@ export function convertLegacyBlocks( name, attributes ) {
 		}
 		name = 'core/embed';
 	}
+
+	// Convert 'core/query-loop' blocks in existing content to 'core/post-template'.
```

Comment:
Do we need to set a timeline for possibly removing this?

Reply:
I'm not sure. Maybe the next version (5.9)?

Reply:
Should this timeline include the addition of `deprecated` call?